core/BUILD

```diff
@@ -26,7 +26,8 @@ ts_library(
         "//common/protos",
         "//common/strings",
         "//protos:ts",
-        "//sqlx",
+        "//sqlx:sqlx",
+        "@npm//@types/node",
```

Comment:
`core` used `@types/node` transitively through `//sqlx`.
 Now `//sqlx:sqlx` is a pure lexer and does not depend on `@types/node`, so I have added here.

Specifically, core need it in places like this
https://github.com/dataform-co/dataform/blob/69cad94ab695d7a0f11d1d227226cc519ea43298/core/utils.ts#L92